lib/tasks/backfill_sponsor_id.rake

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+namespace :in_person_enrollments do
+  desc 'Backfill the sponsor_id column.'
+
+  ##
+  # Usage:
+  #
+  # bundle exec rake in_person_enrollments:backfill_sponsor_id
+  #
+  task backfill_sponsor_id: :environment do |_task, _args|
+    with_timeout do
+      ipp_sponsor_id = IdentityConfig.store.usps_ipp_sponsor_id.to_i
```

Comment:
Check out this [PR](https://github.com/18F/identity-idp/pull/10759/files). The sponsor_id on in_person_enrollments is a string. What do you think about deleting `to_i`?

Reply:
Thanks for catching that!  I made the change in [cbe22ae](https://github.com/18F/identity-idp/pull/10872/commits/cbe22ae08ce7ae9e64cfe6ae645ee5934a309938).